src/main/java/edu/umich/its/cpm/Utils.java

```diff
@@ -234,6 +242,66 @@
 
 		return sessionAttributes;
 	}
+	
+	/**
+	 * login into CTools and become user with sessionId
+	 */
+	protected static HttpContext login_become_admin(Environment env) {
```

Comment:
A HashMap<String, Object> will be returned instead. It shall have two attributes: "sessionId", and "httpContext"